android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/TextareaAnswer.kt

```diff
@@ -14,39 +14,56 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import co.nimblehq.ic.kmm.suv.android.ui.theme.AppTheme
 import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import co.nimblehq.ic.kmm.suv.domain.model.Answer
+import co.nimblehq.ic.kmm.suv.domain.model.AnswerInput
+import co.nimblehq.ic.kmm.suv.domain.model.Answerable
+import co.nimblehq.ic.kmm.suv.domain.model.getContentType
 
+object TextareaAnswerContentDescription {
+    const val TEXTAREA = "TEXTAREA"
+}
+
+@Suppress("NAME_SHADOWING")
 @Composable
 fun TextareaAnswer(
-    onTextChange: (String) -> Unit,
+    answer: Answerable,
+    onInputChange: (AnswerInput) -> Unit,
     modifier: Modifier = Modifier,
-    placeholder: String = "",
+    input: AnswerInput? = null
 ) {
-    var value by remember { mutableStateOf("") }
+    var input by remember { mutableStateOf(input) }
```

Comment:
Instead of having a `Suppress` to ignore the issue, should we rename the param or this val name?

```suggestion
    var input by remember { mutableStateOf(answerInput) }
```

Reply:
Fixed and remove all suppress warning in https://github.com/suho/kmm-ic/pull/123/commits/07f6a3df329b5ed79084a3fe821669529cd47d33